Packs/EDL/Integrations/EDL/README.md

```diff
@@ -1,13 +1,22 @@
 This integration provides External Dynamic List (EDL) as a service for the system indicators (Outbound feed). The feed content can then be sent to 3rd parties for improving their security alignment.
 
+## PAN-OS EDL Management to PAN-OS EDL Service migration steps
+This integration hosts the EDL on the Cortex XSOAR server. Please follow these steps to migrate your EDLs.
+1. Create IOCs in Cortex XSOAR from the EDLs data, which was previously hosted on an external web server. To do so automatically, extract your EDL as a text file from the web server, upload it as a file to the war-room and use the `ExtractIndicatorsFromTextFile` automation. e.g, `!ExtractIndicatorsFromTextFile entryID=<entry_id>` 
```

Comment:
Would rephrase to:
`Convert existing EDL list to indicators in Cortex XSOAR. This can be done automatically: extract your EDL as a text file from the web server it's currently hosted on, upload it as a file to the Playground, and use the...`